### DISCUSSION TITLE
Suggestion: supporting underbar(_)

### DISCUSSION THREAD

**Discussion by danhyun-jeong** [pfrankov/obsidian-colored-tags#24] [Ideas]
I am really enjoying using this plugin!

I would like to make one simple suggestion: currently, if the tag name contains underbars(_), this plugin does not work. It would be great if this plugin would work for tags that contain underbars as well.

Thanks!

**Comment by pfrankov** [comment]
Unfortunately it's a limitation of Obsidian itself.
